transcom/mymove
# B-18477

## [B-18477](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A888485)

## Summary

The Estimated and Actual fields are displaying on the Payment Requests, but the Actual is not populating.
"Actual Weight" is not populated for ppm shipments in the ppm workflow, but rather the weight used for payment and costing is the weight of the truck used full minus the empty weight

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office app
3. Login as a TIO
4. Find PPM in close out status with a payment request
5. verify the actual weight matches the weight that is the full weight of the truck minus the empty weight
6.   if there are multiple weight tickets, it will be the sum of the weights  


### Frontend



## Screenshots
![image](https://github.com/transcom/mymove/assets/136509766/b47deec5-44c5-4f51-959a-af581c0b5bc0)
